source/jquery.flot.touchNavigate.js

```diff
@@ -4,6 +4,9 @@
     'use strict';
 
     var options = {
+        zoom: {
```

Comment:
So we're all clear, this will be a breaking change, but I am fine with this being a breaking change as there should be very few users of this plugin in the flot community as it is extremely new for that user base.

Reply:
Agree.